### ISSUE TITLE
C# XML Doc element `include` is not part of generated documentation

### ISSUE THREAD

**Issue by deap82** [statiqdev/Statiq.Docs#20]
Hi,
I'm trying out Wyam for documenting a C# code base and in the code there is a case where xml doc comments are shared across a few (similiar) classes using the `include` xml doc element, documented here; https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#include

However, when using Wyam with the Docs recipe the contents of these referenced xml files are not included in the Wyam generated docs site. I tried to point to my AssemblyFiles instead of the SourceFiiles, but then I end up with no API item in the main menu at all. I'm not sure if I'm doing it right;
```
Settings[DocsKeys.AssemblyFiles] = "src/**/obj/Debug/**/*.dll";
Settings[DocsKeys.SourceFiles] = null;
```

For the assembly that has these doc comments with `include` elemnets, msbuild also produces an .xml file on the form AssemblyName.xml in the same folder as the .csproj file. This xml file does include the docs referenced from the `include` elements, ie repeated for each time it is referenced.

I'm thinking maybe that file is of interest and in the Wyam docs for the "Docs" recipe it says that features of the recipe includes:

> API documentation from assemblies (with or without XML documentation file from MSBuild).
(https://wyam.io/recipes/docs/overview)

Could someone clarify how to make use of that generated XML documentation as input for Wyam generation?

**Comment by daveaglick** [comment]
@deap82 No, not yet - unfortunately it might be a little while. I'm working furiously on some larger-scale changes to try and get v3 out the door (follow along in the `vnext` branch). In order to focus on that I had to draw a line and put off other issues (otherwise it might never get done). There's a lot of good bugs and small features like this one at the top of the issue list that I'll tackle as soon as v3 is out the door.
